src/entities/RfqmTransactionSubmissionEntity.ts

```diff
@@ -3,11 +3,13 @@ import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
 
 import { BigIntTransformer, BigNumberTransformer } from './transformers';
 
-export enum RfqmTranasctionSubmissionStatus {
+export enum RfqmTransactionSubmissionStatus {
     Submitted = 'submitted',
     Successful = 'successful',
     Reverted = 'reverted',
     DroppedAndReplaced = 'droppedAndReplaced',
+    ConfirmedSuccessful = 'confirmedSuccessful',
+    ConfirmedReverted = 'confirmedReverted',
```

Comment:
Can we document (with a comment maybe?) what is the difference between `Successful` and `ConfirmedSuccessful`?

Reply:
Update coming that should disambiguate